src/main/java/games/strategy/engine/framework/lookandfeel/LookAndFeel.java

```diff
@@ -87,4 +88,40 @@ public static void setupLookAndFeel() {
     });
   }
 
+  private static String getDefaultLookAndFeel() {
+    String defaultLookAndFeel = SubstanceGraphiteLookAndFeel.class.getName();
+
+    if (SystemProperties.isMac()) {
+      // stay consistent with mac look and feel if we are on a mac
+      defaultLookAndFeel = UIManager.getSystemLookAndFeelClassName();
+    }
```

Comment:
I'd suggest using the ternary operator or an if else statement in order to be able to mark the string as final